tsrc/gitlab.py

```diff
@@ -11,6 +11,10 @@
 
 
 class GitLabError(tsrc.Error):
```

Comment:
what is this class useful for? is it used somewhere else?

Reply:
TooManyUsers and GiLlabAPI errors both derive from this class.

This allows fine-grained control for error handling.